### ISSUE TITLE
Proper usage message output

### ISSUE THREAD

**Issue by elfmimi** [project-generator/project_generator#497]
Executing `progen` without any arguments shows error message:
> Traceback (most recent call last):
  File "/home/username/VENV/bin/progen", line 8, in <module>
    sys.exit(main())
  File "/home/username/VENV/lib/python3.9/site-packages/project_generator/__main__.py", line 70, in main
    return args.func(args)
AttributeError: 'Namespace' object has no attribute 'func'

I suppose something like this is required

```python
    def print_usage(args):
        parser.print_usage()
    parser.set_defaults(func=print_usage)
```

before this line.

https://github.com/project-generator/project_generator/blob/34d9d7b4cabd053404e8560c0479cfc6d597d1ae/project_generator/__main__.py#L51

**Comment by 0xc0170** [comment]
Thanks for the report, Ill check

**Comment by 0xc0170** [comment]
@elfmimi I am fixing some other errors today. Feel free to send a pull request, I'll review.

**Comment by 0xc0170** [comment]
Fixed via #501
